exercises/rectangles/example.rs

```diff
@@ -102,29 +110,54 @@ pub fn count(lines: &Vec<&str>) -> usize {
     // The transposed connections have their coordinate system wrong,
     // correct this.
     for l in conns_transposed.lines {
-        conns.lines.insert(Line{x1: l.y1, y1: l.x1, x2: l.y2, y2: l.x2});
+        conns.lines.insert(Line {
+            x1: l.y1,
+            y1: l.x1,
+            x2: l.y2,
+            y2: l.x2,
+        });
     }
     for (p, tcf) in conns_transposed.points {
-        let cf = conns.points.entry(Point{x: p.y, y: p.x}).or_insert(0);
+        let cf = conns.points.entry(Point { x: p.y, y: p.x }).or_insert(0);
         *cf = *cf | (tcf << 2)
     }
 
     let mut total = 0;
-    for (tl_p, tl_cf) in conns.points.iter() { // top left point and connection flags
-        if tl_cf & (CONN_RIGHT|CONN_DOWN) == CONN_RIGHT|CONN_DOWN {
+    for (tl_p, tl_cf) in conns.points.iter() {
+        // top left point and connection flags
+        if tl_cf & (CONN_RIGHT | CONN_DOWN) == CONN_RIGHT | CONN_DOWN {
             for (br_p, br_cf) in conns.points.iter() {
                 // left, right, top, bottom of potential rectangle
                 let l = tl_p.x;
                 let r = br_p.x;
                 let t = tl_p.y;
                 let b = br_p.y;
-                let is_rect =
-                    br_cf & (CONN_LEFT|CONN_UP) == CONN_LEFT|CONN_UP &&
-                    r > l && b > t &&
-                    conns.lines.contains(&Line{x1: l, y1: t, x2: l, y2: b}) &&
-                    conns.lines.contains(&Line{x1: l, y1: t, x2: r, y2: t}) &&
-                    conns.lines.contains(&Line{x1: l, y1: b, x2: r, y2: b}) &&
-                    conns.lines.contains(&Line{x1: r, y1: t, x2: r, y2: b});
+                let is_rect = br_cf & (CONN_LEFT | CONN_UP) == CONN_LEFT | CONN_UP && r > l &&
+                              b > t &&
+                              conns.lines.contains(&Line {
```

Comment:
I'm unsure about this one, the drastic indentation change from line to line might really mess with me